posthog/api/session_recording.py

```diff
@@ -232,8 +232,9 @@ def snapshots(self, request: request.Request, **kwargs):
 
         recording.load_snapshots(limit, offset)
 
-        if not recording.snapshot_data_by_window_id:
-            raise exceptions.NotFound("Snapshots not found")
+        if offset == 0:
```

Comment:
do we need a test?

should we `if blah and not clink:`?

Reply:
This code path is right about to be changed so I'm going to not test and follow up with one in https://github.com/PostHog/posthog/pull/15627